.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,30 @@
+name: Close Stale Issues
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+    inputs:
+      readonly:
+        description: "readonly: Specify true or 1 to prevent changes from being commited to GitHub"
+        default: false
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    env:
+      stale-days: 60
+      close-days: 7
+      stale-label: stale
+    steps:
+    - uses: actions/stale@v4.1.1
+      with:
+        stale-pr-label: $stale-label
+        exempt-issue-labels: stale-exempt
```

Comment:
Is this up-to-date? This field is looking for labels to see if we can exempt them from marking them as stale, I don't think we currently use a `stale-exempt` label.

Reply:
we don't currently have a `stale-exempt` label, but I thought we could start using it. Is there another label you'd recommend?